tests/inputs-test.js

```diff
@@ -35,70 +35,70 @@ const exampleFormData = {
 }
 
 function exampleClick() {
-  return;
+  return
 }
 
 describe('<Button />', function() {
   it ('renders without breaking', function() {
-    const exampleText = 'example button';
-    const button = Enzyme.mount(<Button contents={ exampleText } onClick={ exampleClick } id='button-id' />);
+    const exampleText = 'example button'
+    const button = Enzyme.mount(<Button contents={ exampleText } onClick={ exampleClick } id='button-id' />)
 
-    assert.ok((button).text().indexOf(exampleText) >= 0);
+    assert.ok((button).text().indexOf(exampleText) >= 0)
```

Comment:
I think you can also do...
```
assert.include(button.text(), exampleText)
```

Reply:
Looks like assert doesn't have an include method. https://nodejs.org/api/assert.html

Reply:
Oh, I just assumed you were using chai.

Reply:
Yeah this setup is probably *too* bare-bones. Will be a later update.